### PR TITLE
Versioneer

### PR DESCRIPTION
**Pull request by pelson**
Builds upon #48 to give automatic versioning with versioneer.

### CODE REVIEW

**Comment by rhattersley** [comment]
:+1: Just a couple of PEP8 & future-import failures to fix ... and needs a rebase to get rid of the redundant commit.

**Comment by pelson** [comment]
Thanks @rhattersley. Looking more promising.

File: MANIFEST.in [hunk 1]
```diff
@@ -3,3 +3,5 @@ include *.txt
 recursive-include cf_units *.py
 include cf_units/etc/site.cfg.template
 include CHANGES COPYING COPYING.LESSER INSTALL
+include versioneer.py
+include cf_units/_version.py
```
Comment (rhattersley) [inline MANIFEST.in]: 1. Isn't `cf_units/_version.py` already included by the earlier `recursive-include`?
2. Do we need `setup.cfg` listed in the manifest as well?

Comment (pelson) [inline MANIFEST.in]: > Isn't cf_units/_version.py already included by the earlier recursive-include?

I confess, I let versioneer do this for me automatically. You are right of course.

> Do we need setup.cfg listed in the manifest as well?

I don't believe so, in the same way we do not need `setup.py`.

Comment (pelson) [inline MANIFEST.in]: Updated MANIFEST in the latest commit.

**Comment by rhattersley** [comment]
Thanks @pelson :+1:

**Pull request merged**